tests/integration/targets/azure_rm_subscription/tasks/main.yml

```
@@ -0,0 +1,24 @@
+- name: Get list of all subscriptions
+  azure.azcollection.azure_rm_subscription_info:
```

Everything below needs to be changed like this!

```suggestion
  azure_rm_subscription_info:
```